opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -139,6 +139,109 @@ class SpanKind(enum.Enum):
     CONSUMER = 4
 
 
+class StatusCanonicalCode(enum.Enum):
+    """Represents the canonical set of status codes of a finished Span, following the Standard GRPC codes. See
+    https://github.com/open-telemetry/opentelemetry-specification/blob/2dbae9a491224f1fddfa2bb05c2a1a444c623077/specification/api-tracing.md#statuscanonicalcode.
+    """
+
+    #: Not an error; returned on success.
+    OK = 0
+
+    #: The operation was cancelled, typically by the caller.
+    CANCELLED = 1
+
+    #: Unknown error. For example, this error may be returned when a Status value received from another address space belongs to an error space that is not known in this address space.
+    # Also errors raised by APIs that do not return enough error information may be converted to this error.
+    UNKNOWN = 2
+
+    #: The client specified an invalid argument. Note that this differs from FAILED_PRECONDITION. INVALID_ARGUMENT indicates arguments that are problematic regardless of the state of the system (e.g., a malformed file name).
+    INVALID_ARGUMENT = 2
```

Comment:
Shouldn't this be `3`? https://github.com/grpc/grpc/blob/master/doc/statuscodes.md

Reply:
+1. Looking through the rest and referenced doc, pretty much every code after this is incorrect.

Reply:
I don't think the integer values for the codes are specified for OpenTelemetry, so technically, it wouldn't really be "incorrect". But it would definitely be better if we followed grpc/OpenCensus codes here, because why not.

Reply:
Will add same integer values as grpc, this was a mistake on my side not on purpose